app/main.py

```diff
@@ -1,6 +1,20 @@
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+FORMAT = "json"
+BASE_URL = "http://api.weatherapi.com/v1"
+CITY_NAME = "Paris"
+API_KEY = os.getenv("API_KEY")
```

Comment:
Ensure that the `API_KEY` environment variable is set before running the script. If `API_KEY` is `None`, the request will fail. Consider adding a check to handle this scenario.